src/duiker.py

```diff
@@ -380,7 +380,7 @@ def parse_args(argv):
         PS1="\\u@\\h:\\w$ "
     }}
 
-    PROMPT_COMMAND=__prompt
+    PROMPT_COMMAND='(__prompt $)'
```

Comment:
I don't have a doc reference for that, and it came up in a search for suppressing the "done" output of a backgrounded process. I believe its starting the Python process in a subshell and backgrounding which is either a good or bad idea - not sure which.